apps/desktop/src/renderer/components/Chat/ChatInterface/components/ToolCallBlock/components/AskUserQuestionToolCall/AskUserQuestionToolCall.tsx

```
@@ -158,112 +175,126 @@ export function AskUserQuestionToolCall({
+		Object.keys(answers).length > 0 || answerFallbackText !== undefined;
+
+	// No args available (tool_result-only path with input: {}) — nothing useful to show
+	if (questions.length === 0 && !isCancelledByError && !isCancelledByStop)
+		return null;
+
+	const isAnswered =
+		!isPending && !isCancelledByError && !isCancelledByStop && hasAnswers;
+	const isCancelled =
+		!isPending && !isCancelledByError && !isCancelledByStop && !hasAnswers;
+
+	const answeredQAs = useMemo(
```

Biome (2.4.10)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)